Description: Encode::MIME::HeEader::encode encodes everything with MIME-Q since 2.83.
 Adjust expected output.
Origin: vendor
Bug: https://rt.cpan.org/Public/Bug/Display.html?id=112231
Forwarded: https://rt.cpan.org/Public/Bug/Display.html?id=112231
Bug-Debian: https://bugs.debian.org/825608
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2016-11-06

--- a/t/006-uploads.t
+++ b/t/006-uploads.t
@@ -33,7 +33,7 @@
     is( scalar @parts, 1, "has one part" );
     is $parts[0]->content_type, 'text/plain', 'correct type of the part';
     is $parts[0]->header('Content-Disposition'),
-        'form-data; name="file"; filename="test.txt"',
+        'form-data; name="=?UTF-8?Q?file?="; filename="=?UTF-8?Q?test=2Etxt?="',
         'checked disposition';
 
     is $parts[0]->content, 'stub', 'checked content';
@@ -58,7 +58,7 @@
     is( scalar @parts, 1, "has one part" );
     is $parts[0]->content_type, 'application/octet-stream', 'correct type of the part';
     is $parts[0]->header('Content-Disposition'),
-        'form-data; name="file"; filename="test.txt"',
+        'form-data; name="=?UTF-8?Q?file?="; filename="=?UTF-8?Q?test=2Etxt?="',
         'checked disposition';
 
     is $parts[0]->content, 'stub', 'checked content';
@@ -87,12 +87,12 @@
 
     is $parts[0]->content_type, 'application/octet-stream', 'correct type of the part';
     is $parts[0]->header('Content-Disposition'),
-        'form-data; name="file"; filename="test.txt"',
+        'form-data; name="=?UTF-8?Q?file?="; filename="=?UTF-8?Q?test=2Etxt?="',
         'checked disposition';
     is $parts[0]->content, 'stub', 'checked content';
 
     is $parts[1]->header('Content-Disposition'),
-        'form-data; name="some_arg"',
+        'form-data; name="=?UTF-8?Q?some=5Farg?="',
         'checked disposition';
     is $parts[1]->content, 'some_value', 'checked content';
 }
@@ -117,7 +117,7 @@
 
     is $parts[0]->content_type, 'application/octet-stream', 'correct type of the part';
     is $parts[0]->header('Content-Disposition'),
-        'form-data; name="file"; filename="=?UTF-8?Q?=D1=82=2Ebin?="',
+        'form-data; name="=?UTF-8?Q?file?="; filename="=?UTF-8?Q?=D1=82=2Ebin?="',
         'checked disposition';
     is $parts[0]->content, 'stub', 'checked content';
 }
